pkg/cmd/pkg/package_repo_import.go

```diff
@@ -10,8 +18,49 @@ func newCmdPackageRepoImport(ctx api.Context) *cobra.Command {
 		Use:   "import <repo-file>",
 		Short: "Import a file containing a package repository",
 		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if _, err := os.Stat(args[0]); os.IsNotExist(err) {
+				return fmt.Errorf("path '%s' does not exist", args[0])
+			}
+
+			return nil
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			repoFile, err := os.Open(args[0])
+			if err != nil {
+				return err
+			}
+
+			var repo *dcos.CosmosPackageAddRepoV1Response
+			err = json.NewDecoder(repoFile).Decode(&repo)
+			if err != nil {
+				return err
+			}
+
+			if len(repo.Repositories) == 0 {
+				return errors.New("no repositories found to import")
+			}
+
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+
+			for index, repo := range repo.Repositories {
+				if repo.Name == "" || repo.Uri == "" {
+					fmt.Fprintf(ctx.Out(), "Repo missing name or uri. Skipping.")
+					continue
+				}
+
+				_, err = c.PackageAddRepo(repo.Name, repo.Uri, index)
+				if err != nil {
+					fmt.Fprintf(ctx.ErrOut(), "Error (%s) while adding repo '%s' (%s). Skipping.\n", err, repo.Name, repo.Uri)
+					continue
+				}
+				fmt.Fprintf(ctx.Out(), "Added repo '%s' (%s) at index %d", repo.Name, repo.Uri, index)
```

Comment:
As an informative message, this should go to stderr (see https://github.com/dcos/dcos-cli/blob/master/design/style.md#stdoutstderr).